### DISCUSSION TITLE
Facing Issues in Unlocking Github Discussion

### DISCUSSION THREAD

**Discussion by prembala1710** [community/community#45755] [Discussions]
### Select Topic Area

Bug

### Body

I am having a Github Repository which has a Discussion. I Locked the Discussion to Test a Feature. But It doesn't Change the Lock Conversation to Unlock Conversation. It is a Problem Only for my Repository or is it Occuring for Others too? Also Is there Any Solution for this ?
![image](https://user-images.githubusercontent.com/83074315/215411934-f4c206a2-4683-4e35-9b91-11b722cd3495.png)

**Comment by jge162** [comment]
Hello @prembala1710 sorry to hear the issue, couple things I can think of you can try are the following

- Check repository's branch protection rules.
- Only (check this if you are not repo owner) do you have necessary permissions to unlock the discussion?
- Make sure that the "Issues" option is enabled for the repository.
- Contact [GitHub Support](https://support.github.com/request/account?tags=hubberfy_account) to help you unlock it. 

Choose the topic you feel best suits your issue and submit a ticket for help.

<img width="1034" alt="Screenshot 2023-01-30 at 12 46 45 AM" src="https://user-images.githubusercontent.com/31228460/215429584-ccf91270-e897-44f5-9117-fcfb0f67b5fa.png">

**Comment by prembala1710** [comment]
Thanks for your Suggestion.
I have Created a Github Support Ticket for the Resolving the Issue.

**Reply by jge162** [reply]
No problem. Hope it’s resolved quickly for you.
